### PR TITLE
fix: traditional chinese translations

### PR DESCRIPTION
**Pull request by emmambd**
Authored by @jameslinjl originally in #54 (🎉 - such a big translation review!) , rebased to incorporate recent spec changes.

### CODE REVIEW

File: docs/schedule/reference.zh_TW.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ search:
   
 # GTFS Schedule 參考
 
-**2022 年 12 月 8 日修訂。有關詳細信息，請參閱 [Revision History](../revision-history)。**
+**2022 年 5 月 9 日修訂。有關詳細信息，請參閱 [Revision History](../revision-history)。 **
```
Comment (tzujenchanmbd) [inline docs/schedule/reference.zh_TW.md:8]: Date could be consistent with En page. (En page needs to be revised too)

File: docs/schedule/reference.zh_TW.md [hunk 2]
```diff
@@ -253,328 +251,314 @@ search:
...
-| `destination_id` | 国外身份证参考`stops.zone_id`           | 可选的     | 标识目标区域。如果票价等级有多个目的地区域，请在[fare_rules.txt](#fare_rules.txt)对于每个`destination_id` .<hr/>*示例：`origin_id`和`destination_id`可以一起使用字段来指定票价等级“b”对于区域 3 和 4 之间的旅行有效，对于区域 3 和 5 之间的旅行，[fare_rules.txt](#fare_rules.txt)文件将包含票价等级的这些记录：* <br/>`fare_id,...,origin_id,destination_id` <br/>`b,...,3,4`<br/> `b,...,3,5`                                                                                                                                                                                            |
-| `contains_id`    | 国外身份证参考`stops.zone_id`           | 可选的     | 识别乘客在使用给定票价等级时将进入的区域。在某些系统中用于计算正确的票价等级。<hr/>*示例：如果票价等级“c”与经过 5、6 和 7 区的 GRT 路线上的所有旅行相关联，则[fare_rules.txt](#fare_rules.txt)将包含这些记录：* <br/> `fare_id,route_id,...,contains_id` <br/>  `c,GRT,...,5` <br/>`c,GRT,...,6` <br/>`c,GRT,...,7` <br/> *因为所有`contains_id`必须匹配区域才能应用票价，经过 5 区和 6 区但不经过 7 区的行程将没有票价等级“c”。有关更多详细信息，请参阅[ https://code.google.com/p/googletransitdatafeed/wiki/FareExamples](https://code.google.com/p/googletransitdatafeed/wiki/FareExamples)在 GoogleTransitDataFeed 项目 wiki 中。* |
-
-### fare_media.txt
-
-文件：**可選**
-
-主鍵 ( `fare_media_id` )
```
Comment (tzujenchanmbd) [inline docs/schedule/reference.zh_TW.md:365]: Seems all fare_media parts have been removed

**Review by tzujenchanmbd — COMMENTED** [review]
- Confirmed that all simplified Chinese characters are replaced by traditional characters.
- Found 1 significant change at `fare_media`.
- We may merge this commit first and modify later?

**Comment by fredericsimard** [comment]
@Sergiodero Should we keep this around? Chances are low that this would still be relevant IMO.

**Comment by Sergiodero** [comment]
> @Sergiodero Should we keep this around? Chances are low that this would still be relevant IMO.

I agree with you since we'll get a new translated version, but maybe we should just double check with @tzujenchanmbd first.

**Comment by Sergiodero** [comment]
After discussing it, we'll move on to close this PR as these changes will no longer be relevant to the new version of [GTFS.org](GTFS.org), Chinese translations for this new version will be added soon.

**Pull request closed without merging**